src/main.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { render as reactDomRender } from 'react-dom'; 
+import superagent from 'superagent';
+import './styles/main.scss';
+
+const apiUrl = 'http://www.reddit.com/r/'; // ??? 
+
+// onSubmit the form should make a request to reddit
+// it should make a get request to http://reddit.com/r/${searchFormBoard}.json?limit=${searchFormLimit}
+// on success it should pass the results to the application state
+// on failure it should add a class to the form called error and turn the form's inputs borders red
+
+class RedditSearchForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      redditForum: 'travel',
+      redditForumLimit: '1',
+    };
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleForumChange = this.handleForumChange.bind(this);
+    this.handleLimitChange = this.handleLimitChange.bind(this);
+  }
+
+  handleForumChange(event) {
+    this.setState({ redditForum: event.target.value }, () => {
+      console.log(this.state);
+    });
+  }
+
+  handleLimitChange(event) {
+    this.setState({ redditForumLimit: event.target.value }, () => {
+      console.log(this.state);
+    });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    // this.props.redditForum(this.state.redditForum);
+  }
+  redditForumSelect() {
```

Comment:
This function should have an arity of two, one for `topic` and one for `limit`

Also, consider moving this function into your `App` component, then you can `setState` from the results of your reddit search to your app and pass them as props into the `RedditSearchResultsList` component